x/programs/rust/wasmlanche-sdk/src/state.rs

```diff
@@ -149,8 +149,8 @@ impl<'a, K: Key> State<'a, K> {
         }
 
         #[derive(BorshSerialize)]
-        struct PutArgs<Key> {
-            key: Key,
+        struct PutArgs<K> {
+            key: K,
```

Comment:
This change isn't necessary

Reply:
I found that it was more correct than repeating the trait, since we probably don't want to have a different key in the `PutArgs` and in the actual state, I reverted it though